zeebe/logstreams/src/main/java/io/camunda/zeebe/logstreams/impl/flowcontrol/RateLimit.java

```diff
@@ -25,4 +38,25 @@ public RateLimiter limiter() {
     }
     return RateLimiter.create(limit, rampUp);
   }
+
+  public record Throttling(
```

Comment:
❓ should we somehow assert that the throttling min rate is not higher than the rate limit if these are enabled?

Reply:
Good point, I added it here: ae02649